src/Sylius/Behat/Context/Api/Admin/ManagingShippingMethodsContext.php

```diff
@@ -123,6 +220,28 @@ public function iTryToCreateANewShippingMethodWithValidData(): void
         ]);
     }
 
+    /**
+     * @When I do not specify amount for :calculatorName calculator
+     */
+    public function iDoNotSpecifyAmountForCalculator(string $calculatorName): void
+    {
+        $channelCode = $this->sharedStorage->get('channel')->getCode();
+        match ($calculatorName) {
+            'Flat rate per shipment' => $this->client->addRequestData('calculator', DefaultCalculators::FLAT_RATE),
+            'Flat rate per unit' => $this->client->addRequestData('calculator', DefaultCalculators::PER_UNIT_RATE),
+        };
+
+        $this->client->addRequestData('configuration', [$channelCode => ['amount' => null]]);
```

Comment:
```suggestion
        match ($calculatorName) {
            'Flat rate per shipment' => $this->client->addRequestData('calculator', DefaultCalculators::FLAT_RATE),
            'Flat rate per unit' => $this->client->addRequestData('calculator', DefaultCalculators::PER_UNIT_RATE),
        };

        $channelCode = $this->sharedStorage->get('channel')->getCode();
        $this->client->addRequestData('configuration', [$channelCode => ['amount' => null]]);
```
You could also throw on the default match since no known calculator has been passed.